cloudgarden-pwa/src/auth.tsx

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
This file may actually be unecessary, created it when following along with this tutorial https://developer.okta.com/blog/2019/03/06/simple-user-authentication-in-react